content/articles/2019-05-27-Bonding-period-ends.md

```diff
@@ -0,0 +1,82 @@
+Title: Coding phase starts!
+Date: 2019-05-27 11:30
+Category: GSCOC
+Tags: GSOC, GSOC19, poliastro
+slug: 2019-05-27-poliastro-gsoc-jorge
+lang: en
+Author: Jorge Martínez
```

Comment:
```suggestion
Author: Jorge Martínez Garrido
```

To make it the same to https://github.com/poliastro/poliastro.github.io/blame/sources/content/articles/2019-05-10-poliastro-gsoc-jorge.md